src/components/StoryPointsContainer.tsx

```diff
@@ -0,0 +1,79 @@
+'use client'
+
+import { socketRoomEmitter } from '@/services/socket'
+import { useSocketListener } from '@/services/socket'
+import { useState, useEffect } from 'react'
+import StoryPointButton from './StoryPointButton'
+import type { RoomInfoData } from './RoomMainUi'
+
+// export default function StoryPointsContainer({
+// 	setSelectedStoryPoint,
+// }: { setSelectedStoryPoint: (value: string) => void }) {
+export default function StoryPointsContainer({
+	roomId,
+	userName,
+}: RoomInfoData) {
+	const [selectedStoryPoint, setSelectedStoryPoint] = useState<string | null>(
+		null,
+	)
+	const [isPointBtnDisabled, setIsPointBtnDisabled] = useState(false)
+
+	// const resetStoryPointBtns = useSocketListener('reset-points')
+	const resetStoryPointBtns = useSocketListener('show-disable-reset-points')
```

Comment:
todo/bug: Another usecase where you are miss using custom hooks and effects. We'll need to talk about this :)

Reply:
DONE. New commit for the code.